code/modules/virus2/analyser.dm

```diff
@@ -94,7 +94,7 @@
 
 /obj/machinery/disease2/diseaseanalyser/Topic(href, href_list)
 	if(..())
-		return 1
+		return
```

Comment:
Oh this should still be return 1, that way it still returns for any future children of this machine